web_monitoring/diffing_server.py

```diff
@@ -74,16 +74,23 @@ def __init__(self, url, body, headers):
 
 access_control_allow_origin_header = \
     os.environ.get('ACCESS_CONTROL_ALLOW_ORIGIN_HEADER')
-
+ALLOWED_ORIGINS = set()
+if access_control_allow_origin_header:
+    ALLOWED_ORIGINS = set([orig.strip() for orig in
```

Comment:
Please avoid abbreviations, especially when they can be ambiguous for many people (i.e. `orig` → `origin`).

Reply:
Done.